src/test/rustdoc/process-termination_nostd.rs

```diff
@@ -0,0 +1,19 @@
+// compile-flags:--test
+
+#![no_std]
```

Comment:
`#![no_std]` has to be in the doctest itself otherwise we're not actually testing anything new.